SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -1456,8 +1461,13 @@ bool CreateCustomAuthoritySet(FString SchemaInputPath, FString SchemaOutputPath,
 			return true;
 		});
 
+		// Put the generated schema binary for our custom schema in the Intermediate folder.
+		// This is so it doesn't stomp the users (or tests) proper schema binary used by the runtime.
+		FString SchemaCompilerIntermediateDir = FPaths::Combine(FPaths::ProjectIntermediateDir(), TEXT("Improbable"), TEXT("schema"));
```

Comment:
If I understand correctly, the test didn't even use this schema so it doesn't matter that it's in this random folder, right?
Just an intermediate step that was stomping real things?

Reply:
Yup, it doesn't use the schema but it uses the produced json